### PR TITLE
(MAINT) Upgrade beaker and sqa-utils gems.

### PR DESCRIPTION
**Pull request by waynr**
I cannot run beaker locally on my laptop without upgrading to 2.2. I am currently working to test this out on an experimental pipeline.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.delivery.puppetlabs.net/
```
Comment (elyscape) [inline Gemfile.lock]: Be advised that this does not appear to be accessible to the internet at large.

**Comment by waynr** [comment]
@elyscape Thanks for pointing that out, it is now fixed.

File: Gemfile [hunk 2]
```diff
@@ -5,8 +5,8 @@ gem 'jira-ruby', :group => :development
 
 group :test do
   gem 'rspec'
-  gem 'beaker', '~>1.17.0'
+  gem 'beaker', '~> 2.2'
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils'
+    gem 'sqa-utils', '> 0.11'
```
Comment (rlinehan) [inline Gemfile]: Do we need to pin this to > 0.11, or can we just `bundle update` to update the Gemfile.lock?

**Comment by rlinehan** [comment]
Aside from my comment on whether we need to pin sqa-utils, I'm +1 on this. Couldn't run acceptance tests locally without beaker 2.2 - I was getting a weird stacktrace otherwise.

File: Gemfile (hunk 2 above)
Comment (waynr) [inline Gemfile]: Hmm, I think I intended for this to be `~> 0.11` which I believe would prevent major version updates while still giving us minor and patch version updates.

Comment (rlinehan) [inline Gemfile]: Okay, I guess that makes more sense if we don't want to have automatic major version updates.

**Comment by waynr** [comment]
@rlinehan 
Okay I updated the version constraing on `sqa-utils` so as far as I'm concerned it's ready to go.

**Comment by rlinehan** [comment]
:+1:

**Comment by cprice404** [comment]
@rlinehan does your other PR subsume this one?

**Comment by rlinehan** [comment]
@cprice404 yeah, my pr has this same stuff in it because I just wanted to push it up before lunch, but I'm happy merging this and then removing my commit from my PR.

**Comment by cprice404** [comment]
I'd say let's just close this and merge yours.

**Pull request closed without merging**